src/main/java/org/spongepowered/mod/mixin/core/event/player/MixinEventPlayerInteractBlock.java

```diff
@@ -79,4 +85,15 @@ public EntityInteractionType getInteractionType() {
         return Optional.fromNullable(new Cause(null, this.entityPlayer, null));
     }
 
+    @Override
+    public Optional<Vector3d> getClickedPosition() {
+        return Optional.of(VecHelper.toVector(this.pos).toDouble());
```

Comment:
Forget what I said above. I think I might have founded it. But ehm:
```java
@Redirect(method = "processPlayerBlockPlacement", at = @At(value = "INVOKE", target = "Lnet/minecraftforge/event/ForgeEventFactory;onPlayerInteract(Lnet/minecraft/entity/player/EntityPlayer;Lnet/minecraftforge/event/entity/player/PlayerInteractEvent$Action;Lnet/minecraft/world/World;Lnet/minecraft/util/BlockPos;Lnet/minecraft/util/EnumFacing;)Lnet/minecraftforge/event/entity/player/PlayerInteractEvent;"))
public PlayerInteractEvent onC08(EntityPlayer player, PlayerInteractEvent.Action action, World world, BlockPos pos, EnumFacing face, C08PacketPlayerBlockPlacement packetIn) {
    System.out.println(new javax.vecmath.Vector3d(packetIn.getPlacedBlockOffsetX(), packetIn.getPlacedBlockOffsetY(), packetIn.getPlacedBlockOffsetZ()));
    return ForgeEventFactory.onPlayerInteract(player, action, world, pos, face);
}
```
```
[12:35:23] [Server thread/INFO] [STDOUT]: [net.minecraft.network.NetHandlerPlayServer:onC08:88]: (0.0, 0.0, 0.0)
[12:35:27] [Server thread/INFO] [STDOUT]: [net.minecraft.network.NetHandlerPlayServer:onC08:88]: (0.0, 0.0, 0.0)
[12:35:27] [Server thread/INFO] [STDOUT]: [net.minecraft.network.NetHandlerPlayServer:onC08:88]: (0.0, 0.0, 0.0)
[12:35:28] [Server thread/INFO] [STDOUT]: [net.minecraft.network.NetHandlerPlayServer:onC08:88]: (0.0, 0.0, 0.0)
[12:35:28] [Server thread/INFO] [STDOUT]: [net.minecraft.network.NetHandlerPlayServer:onC08:88]: (0.0, 0.0, 0.0)
[12:35:28] [Server thread/INFO] [STDOUT]: [net.minecraft.network.NetHandlerPlayServer:onC08:88]: (0.0, 0.0, 0.0)
[12:35:29] [Server thread/INFO] [STDOUT]: [net.minecraft.network.NetHandlerPlayServer:onC08:88]: (0.0, 0.0, 0.0)
[12:35:29] [Server thread/INFO] [STDOUT]: [net.minecraft.network.NetHandlerPlayServer:onC08:88]: (0.0, 0.0, 0.0)
```
The data the client sends when that event occurs isn't very interesting.